### PR TITLE
create valid pem on windows and linux

### PR DESCRIPTION
**Pull request by kramvan1**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ $ git clone https://github.com/jjasghar/chef-openstack-testing-stack.git testing
 $ cd testing-stack
 $ vi vagrant_linux.rb # change the 'vm.box' to the box you'd like to run.
 $ chef exec rake berks_vendor
-$ chef exec ruby -e "require 'openssl'; puts OpenSSL::PKey::RSA.new(2048).to_pem" > .chef/validator.pem
+$ chef exec ruby -e "require 'openssl'; File.binwrite('.chef/validator.pem', OpenSSL::PKey::RSA.new(2048).to_pem)"
```
Comment (cmluciano) [inline README.md]: Should this still be piped to .chef/validator.pem?

Comment (kramvan1) [inline README.md]: This is using the File.write, so it goes directly to the file, no piping needed.

Comment (cmluciano) [inline README.md]: Totally misread the code. Testing now.

**Comment by cmluciano** [comment]
+1 this works on OS X Yosemite

**Pull request merged**